src/elements/content-explorer/MetadataViewContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { MetadataView, type MetadataViewProps } from '@box/metadata-view';
+
+import type { SortDescriptor } from 'react-aria-components';
+import type { Collection } from '../../common/types/core';
+
+export interface MetadataViewContainerProps extends Omit<MetadataViewProps, 'items'> {
+    currentCollection: Collection;
+    onSortChange?: (sortBy: string, sortDirection: string) => void;
+}
+
+const MetadataViewContainer = ({
+    actionBarProps,
+    columns,
+    currentCollection,
+    onSortChange,
+    tableProps,
+    ...rest
+}: MetadataViewContainerProps) => {
+    const { items = [], metadataTemplate } = currentCollection;
```

Comment:
metadataTemplate seems like it should be part of outer props rather than in the current collection